tests/visual/window-rtl.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+<html lang="en" class="k-typography k-no-animations">
+<head>
+    <title>Window</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" data-role="kendo-theme" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            grid-template-columns: repeat(3, 1fr);
+            align-items: start;
+        }
+        /* test page overrides */
+         .k-window {
+            position: relative;
+        }
+    </style>
+</head>
+<body dir="rtl" class="k-rtl">
+
+    <div id="test-area" class="grid">
+        <div class="k-widget k-window k-window-wrapper">
```

Comment:
There is missing `dir="rtl"` attribute in the tests. The `dir="rtl"` is rendered from Angular components on the host element to specify that the component will be styled corresponding to RTL mode.